src/atoms/Elbil24Logo.js

```diff
@@ -0,0 +1,78 @@
+import React  from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+import { getColor } from '../utils';
+
+const Svg = styled.svg`
+  box-sizing: border-box;
+	
+	& .primary {
+		fill: ${props => (props.pro ? getColor('white') : getColor('secondary'))}
+	}
+
+	& .secondary {
+		fill: ${props => (props.pro ?getColor('secondary') : getColor('primary'))}
+	}
+`;
+
+/* eslint-disable max-len */
+const Elbil24Logo = props => (
+	<Svg
+		role="img"
+		aria-labelledby="title"
+		id="Elbil24Logo"
```

Comment:
HTML restricts the same ID to be used only once per page. Remove it.

Add a `<title>Elbil24</title>` inside the Svg, comparable to `alt="Elbil24"` on img tags.